packages/enzyme-test-suite/test/Adapter-spec.jsx

```diff
@@ -250,6 +254,56 @@ describe('Adapter', () => {
       }));
     });
 
+    itIf(is('>= 16'), 'shallow renders react portals', () => {
+      const options = { mode: 'shallow' };
+      const renderer = adapter.createRenderer(options);
+      const innerDiv = <div className="Foo">Hello World!</div>;
+      const containerDiv = { nodeType: 1 };
+      const Foo = () => (
+        createPortal(
+          innerDiv,
+          containerDiv,
+        )
+      );
+
+      renderer.render(<Foo />);
+
+      const node = renderer.getNode();
+
+      const { rendered: { props: { children } } } = node;
+      expect(children).to.equal(innerDiv);
+
+      cleanNode(node);
+
+      expect(prettyFormat(node)).to.equal(prettyFormat({
+        nodeType: 'function',
+        type: Foo,
+        props: {},
+        key: undefined,
+        ref: null,
+        instance: null,
+        rendered: {
+          nodeType: 'portal',
+          type: Portal,
+          props: {
+            containerInfo: containerDiv,
+          },
+          key: undefined,
+          ref: undefined,
```

Comment:
based on your comment, it seems like this should be `null`, and not `undefined`. Thoughts?

Reply:
Yep, does it make sense that portal `rendered.ref` should always be null? Im not sure what refs are here. I will follow up on the other issue (array wrapping `rendered` in shallow) as well but separately.

Reply:
Does ref ever return `undefined` for anything else? If not, then it should probably be null here.